quinn-proto/src/endpoint.rs

```diff
@@ -415,11 +409,13 @@ impl Endpoint {
         version: u32,
         addresses: FourTuple,
         crypto: &Keys,
-        src_cid: &ConnectionId,
-        dst_cid: &ConnectionId,
+        first_decode: &PartialDecode,
```

Comment:
I suppose this change could be made in the initial commit?

Reply:
Mooted by other refactoring.